sql/migrations/20210515110157_world.sql

```diff
@@ -0,0 +1,851 @@
+(3224,3,-2260.51,-306.049,-9.29993,100,0,0,0),
+(3224,4,-2257.75,-312.749,-9.29993,100,5000,5,0),
+(3224,5,-2265.13,-312.055,-9.42604,100,0,0,0),
+(3224,6,-2272.79,-315.722,-9.42604,100,5,5000,0),
+(3224,7,-2270.73,-308.69,-9.42604,100,0,0,0),
+(3224,8,-2274.7,-300.973,-9.42604,100,5000,5,0),
+(3224,9,-2270.95,-301.636,-9.29993,100,0,0,0),
+(3224,10,-2267.42,-302.304,-9.29993,100,0,0,0),
+(3224,11,-2263.55,-300.676,-9.29993,100,0,0,0),
+(3224,12,-2259.32,-296.491,-9.34905,100,5000,5,0);
+
+-- Explosive Sheep 2675 (credit cmangos)
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (267501, 2675, 0, 11, 0, 100, 0, 0, 0, 0, 0, 267501, 0, 0, 'Explosive Sheep - Cast Explosive Sheep Passive and Stealth Detection on Spawn');
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (267501, 0, 15, 4051, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Explosive Sheep - Cast Spell');
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (267501, 0, 15, 8279, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Explosive Sheep - Cast Spell');
+
+-- Naga Explorer 1907 (credit cmangos)
+INSERT INTO `creature` (`guid`, `id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecsmin`, `spawntimesecsmax`, `wander_distance`, `health_percent`, `movement_type`, `patch_max`) VALUES
+(2090, 1907, -13669.1, 616.477, 7.57179, 3.58152, 300, 300, 5, 100, 1, 10),
+(2093, 1907, -13701, 633.343, 7.3798, 3.06592, 300, 300, 5, 100, 1, 10),
+(2554, 1907, -13705.4, 574.627, 18.7638, 4.46864, 300, 300, 5, 100, 1, 10),
+(180, 1907, -13783.5, 645.277, 14.8763, 5.95696, 300, 300, 5, 100, 1, 10);
+UPDATE `creature` SET `position_x` = -13780.404297, `position_y` = 498.528076, `position_z` = 69.540359, `wander_distance` = 5, `movement_type` = 1 WHERE `guid` = 1966;
+
+-- Tethis 730 (credit cmangos)
+INSERT INTO `creature` (`guid`, `id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecsmin`, `spawntimesecsmax`, `wander_distance`, `health_percent`, `movement_type`, `patch_max`) VALUES
+(2588, 730, -12978.6, 411.841, 21.7456, 3.43503, 420, 600, 15, 100, 1, 10),
+(565, 730, -13017.2, 193.101, 22.7814, 1.45268, 420, 600, 15, 100, 1, 10);
+UPDATE `creature` SET `wander_distance` = 20, `movement_type` = 1 WHERE `id` = 730;
+DELETE FROM `pool_creature_template` WHERE `id` = 730;

```

Comment:
Don't use the DELETE statement if there is nothing to delete. There is no data with id=730 in the table currently. This is supposed to express intent, if you have a DELETE in your sql, it must mean there was something in there before and you intend to remove it on purpose because it was wrong. Otherwise you just need an INSERT with the new data.